src/dawn/engine_dawn.cc

```diff
@@ -417,64 +437,52 @@ Result EngineDawn::DoBuffer(const BufferCommand*) {
   return Result("Dawn:DoBuffer not implemented");
 }
 
-Result EngineDawn::CreateCommandBufferBuilderIfNeeded() {
-  if (command_buffer_builder_)
-    return {};
-
-  // Create the command buffer builder because it doesn't exist yet.
-  if (!device_) {
-    return Result(
-        "EngineDawn: Can't create command buffer builder: device is not "
-        "initialized");
-  }
-  command_buffer_builder_ = device_->CreateCommandEncoder();
-  if (command_buffer_builder_)
-    return {};
-  return Result("EngineDawn: Can't create command buffer builder");
+Result EngineDawn::CreateRenderObjectsIfNeeded(
+    RenderPipelineInfo* render_pipeline) {
+  return CreateFramebufferIfNeeded(render_pipeline);
```

Comment:
Do we expect to do more here? Or else, why not just call CreateFramebufferIfNeeded directly to make it clearer whta's happening?